californium-core/src/main/java/org/eclipse/californium/core/coap/OptionSet.java

```diff
@@ -165,9 +166,9 @@ public OptionSet(OptionSet origin) {
 		if (origin.block2 != null)
 			block2          = new BlockOption(origin.block2);
 		
-		observe = origin.observe;
+		observe = origin.observe;	
 		if(origin.oscore != null)
-			oscore	= origin.oscore.clone();
+			oscore			= origin.oscore.clone();
```

Comment:
Please, try to omit changing files only for formatting.

Reply:
I will remove this.